Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/SDKProjectTemplate.g.targets.template

```diff
@@ -25,7 +25,12 @@
 <!--SUPPORTED_PLATFORM_BUILD_CONDITION_TEMPLATE_END-->
   </Target>
 
-  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-    <Exec Command="xcopy /y $(TargetDir)$(TargetName).dll $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\&#xD;&#xA;xcopy /y $(TargetDir)$(TargetName).pdb $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\" />
+  <ItemGroup>
+    <_MSB4UPublishFiles Include="$(TargetDir)$(TargetName).dll"/>
+    <_MSB4UPublishFiles Include="$(TargetDir)$(TargetName).pdb"/>
+  </ItemGroup>
```

Comment:
NIT: In most normal cases I would expect TargetDir and TargetName to be defined before any targets run, but to be safe/consistent with the previous behavior you can move the ItemGroup to inside the CopyPublishFiles target so it is not evaluated until that point in the build process.

Reply:
Good call!